### DISCUSSION TITLE
About multiple commits are prompts for this problem

### DISCUSSION THREAD

**Discussion by xc-home** [community/community#57524] [GitHub Education]
### Hi everyone,

Now the main hint of the following picture problem, how should I do ah, I hope someone can give me a clear way

![Uploading Snipaste_2023-06-08_18-08-35.jpg…]()

**Comment by xc-home** [comment]
![Snipaste_2023-06-08_18-08-35](https://github.com/community/community/assets/76017772/6223fcc1-68e0-4e00-b83a-72282fbf3dde)

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open.
